sites/dotnet-guide/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "dotnet-guide",
+  "description": "Gatsby-driven compendium of GoLand learning resources",
+  "version": "1.0.0",
+  "author": "Maarten Balliauw <maarten@jetbrains.com>",
+  "keywords": [
+    "gatsby, typescript"
+  ],
+  "license": "Apache-2.0",
+  "dependencies": {
+    "gatsby": "2.15.14",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "build": "gatsby build --prefix-paths",
+    "clean": "node ../../rimraf-standalone.js .cache/* public/*",
```

Comment:
As a note, Gatsby now has a "gatsby clean" command that does the same thing as this, possibly better, if you want to switch to it.

Reply:
Feel free to update cross-repo?